examples/09_Kotlin_JS/02_js_function.md

```diff
@@ -5,6 +5,19 @@ Should be used with extreme care.
 
 <div class="language-kotlin" theme="idea" data-min-compiler-version="1.3" data-target-platform="js">
 
+```kotlin
```

Comment:
from the text above ^^:

You can inline JavaScript code into your Kotlin code using the `js("…")` function.
 This should be used with extreme care.